rosdep-install.sh

```diff
@@ -9,12 +9,12 @@ EXIT_STATUS=1
 COUNT=0
 while [ $EXIT_STATUS == 1 -a $COUNT -lt 3 ] ; do  # try 3 times with "Continue installing despite errors." option
     COUNT=$((COUNT + 1))
-    rosdep install -q -y --rosdistro $ROS_DISTRO $ROSDEP_ADDITIONAL_OPTIONS --from-paths ${ROS_PACKAGE_PATH_REVERSED} .
+    rosdep install -q -y --rosdistro $ROS_DISTRO $ROSDEP_ADDITIONAL_OPTIONS --from-paths ${ROS_PACKAGE_PATH_REVERSED} . | grep -v -e '\(Preparing to unpack\|Unpacking\)'
     EXIT_STATUS=$?
```

Comment:
need to use PIPESTATUS instead of `$?`

https://unix.stackexchange.com/questions/14270/get-exit-status-of-process-thats-piped-to-another

Reply:
thanks fixed.